content/content-zh-cn.md

```diff
@@ -0,0 +1,449 @@
+傅里叶变换是一种在各个领域都经常使用的数学工具。这个网站将为你介绍傅里叶变换能干什么，为什么傅里叶变换非常有用，以及你如何利用傅里叶变换干漂亮的事。就像下面这样：
+
+<canvas id="self-draw" class="sketch" width=500 height=500></canvas>
+
+我将为你解释这个动画是如何工作的，沿途为你详细地解释傅里叶变换！
+
+这次旅途结束后，你将会掌握下面这些知识：
+- 什么是傅里叶变换
+- 傅里叶变换的一些实际用途
+- 傅立叶变换的一些很酷的用法（虽然有些没有实际意义）
+
+我们现在暂时不提那些复杂的数学公式。傅里叶背后的数学原理十分有趣，但最好还是先从它的实际应用开始，以及为什么要使用它。 如果你想了解更多，下面提供了一些进一步的阅读建议！
+
+## 傅里叶变换是什么
+
+简而言之，傅里叶变换把一个输入信号分解成一堆正弦波的叠加。就像大多数数学方法一样，这个名字来自一个名叫傅立叶的人。
+
+让我们从一些简单的例子开始，然后继续前进。首先，我们来看看什么是波 —— 波随着时间的推移，一直按照某一规律变化。
+
+这是一个波的例子：
+
+<canvas id="combo-sine-wave" class="sketch" width=500 height=300></canvas>
+
+这个波可以分解为两个正弦波的叠加。也就是说，当我们将两个正弦波相加时，就会得到原来的波。
+
+<canvas id="combo-sine-wave-split" class="sketch" width=500 height=500></canvas>
+
+傅里叶变换可以让我们从一个复杂的波形里面，把构成这个波的单个正弦波分离出来。在这个例子中，你几乎可以通过“脑补”完成这一操作。
+
+为什么？事实证明，现实世界中的许多事物间的互相交互，都是基于正弦波。我们通常将这种波的快慢的性质，称为波的频率。
+
+最明显的例子就是声音 —— 当我们听到声音时，我们听不到那条波浪线，但我们听到构成声音的正弦波的不同频率。
+
+<button id="together-button" class="button">Play Full Wave</button>
```

Comment:
So there are some buttons that also need to be translated, I'll leave comments on the lines that still have some English text.

Here it's "Play Full Wave"

Reply:
Wow, sorry for that, I will make an update